### ISSUE TITLE
Poll sites occasionally to update info (web bridge)

### ISSUE THREAD

**Issue by FurblandChannel** [snarfed/bridgy-fed#1485]
On the web bridge, site descriptions and/or favicons become outdated quite fast. Perhaps the bridge could check monthly (starting a month after the bridge account was created) and update any data if changes occur. This might also be a useful feature for fediverse to at bridging (and vice versa, of course).

**Comment by snarfed** [comment]
Yes! This would be nice. Right now, web users can click the 🔁 button on their user page to update their profile, or send a webmention from their home page. https://fed.brid.gy/docs#update-profile

This works for fediverse and Bluesky accounts too, but those protocols already send profile updated to Bridgy Fed automatically.
